mkdocs.yml

```diff
@@ -102,21 +103,30 @@ nav:
   - Resources:
     - Cloud Resources: Resources/Cloud_Resources/index.md
     - FPGA Resource: Resources/FPGA_Resources/index.md
-    - GPU Resources: Resources/GPU_Resources/index.md
+    - GPU Resources:
+      - Overview: Resources/GPU_Resources/index.md
+      - CMS Resources:
+        - SWAN: Resources/GPU_Resources/CMS_Resources/SWAN.md
+        - LXPLUS Condor: Resources/GPU_Resources/CMS_Resources/lxplus_condor.md
+        - LXPLUS GPU: Resources/GPU_Resources/CMS_Resources/lxplus_gpu.md
+        - ml.cern.ch: Resources/GPU_Resources/CMS_Resources/ml_cern_ch.md
+        - Notebooks:
+          - PytorchMNIST: Resources/GPU_Resources/CMS_Resources/Notebooks/PytorchMNIST.md
+          - TopTaggingMLP: Resources/GPU_Resources/CMS_Resources/Notebooks/TopTaggingMLP.md
   - Tutorials:
     - Optimization:
       - Model optimization: optimization/model_optimization.md
       - Feature importance: optimization/importance.md
       - Data augmentation: optimization/data_augmentation.md
-    - Validation:
+    - General Advice:
       - Introduction: validation/intro.md
       - Before training:
         - Domains: validation/before/domains.md
         - Features: validation/before/features.md
         - Inputs: validation/before/inputs.md
         - Model: validation/before/model.md
         - Metrics & Losses: validation/before/metrics.md
-      - Throughout training: 
+      - During training:
```

Comment:
@cmadrid1 We talked about this in the past. I don't have any objection to using the term "During." Do you?

Reply:
I like it.